README.md

```diff
@@ -18,7 +18,8 @@ We try to maintain a list of all the stations in Belgium using CSV so everyone c
  * `alternative-de`: alt. name in German, if available
  * `alternative-en`: alt. name in English, if available
  * `country-code`: the code of the country the station belongs to
-
+ * `vehicle_frequency`: the relative frequency (0 .. 1) of the amount of trains stopping in this station.
```

Comment:
Sorry for changing my mind, but I'd rather see the same meaning used as at st:avgStopTimes (http://semweb.mmlab.be/ns/stoptimes#avgStopTimes). I think it's not that hard to calculate an average stoptimes a day instead of a vehicle frequency between 0 and 1? Or am I wrong?